requirements.txt

```diff
@@ -20,7 +20,7 @@ django-model-utils==4.0.0
 babel==2.8.0
 django-import-export==2.0.2
 django-money==1.1
-
+stripe
```

Comment:
In the requirements files its important to always specify a specific version number, it allow us to:

 - easily reproduce specific state (for error and easy installation)
 - keep track of versions and manage update properly
 - block the version to an old one the time we implement potential fix in case of breaking changes